src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/TestRunner.cs

```diff
@@ -384,6 +384,10 @@ internal static void TestToString(FormulaValue result, StringBuilder sb)
             {
                 sb.Append($"GUID(\"{gv.Value}\")");
             }
+            else if (result is ColorValue cv)
+            {   
+                sb.Append($"Color [A={cv.Value.A}, R={cv.Value.R}, G={cv.Value.G}, B={cv.Value.B}]");
```

Comment:
We try, as much as possible, to have valid PowerFx expressions as the result of tests. In this case we can use the RGBA function, something along the lines of

    sb.Append($"RGBA({cv.Value.R},{cv.Value.G},{cv.Value.B},{Math.Round(cv.Value.A / 255.0, 3)})");

Reply:
Yeah so the string you see here is what I saw in the REPL example, which is using ToString() on the Color object... But I see what you are saying and that makes sense. Should it be RGBA() or ColorValue() ?

Reply:
Both would work, as long as we have a valid Power Fx expression. Your call :-)

Reply:
RGBA it is - seemed more readable